addons/binding/org.openhab.binding.amazondashbutton/README.md

```diff
@@ -78,6 +78,7 @@ Follow the instructions to set up the Dash Button in their mobile app.
 When you get to the step where it asks you to pick which product you want to map it to, just quit the setup process.
 
 {: #no-phonehome}
+
```

Comment:
No. This is an exception we should teach the linter by submitting a PR.
This is a "block inline attribute" defined for Kramdown https://kramdown.gettalong.org/quickref.html#block-attributes
I am almost certain that it has to be next to the headline element without an empty line.